source/stylesheets/components/typography.css

```
@@ -28,6 +28,11 @@
 }
 
 /* paragraphs */
+.par-2xl {
+  @apply text-[40px] leading-[48px] md:text-[64px] md:leading-[72px] font-bold;
+  -webkit-text-stroke: 1px currentColor;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)